components/script/dom/bindings/buffer_source.rs

```diff
@@ -418,7 +418,7 @@ where
     /// without causing conflicts , unexpected behavior.
     /// <https://github.com/servo/mozjs/blob/main/mozjs-sys/mozjs/js/public/ArrayBuffer.h#L89>
     unsafe extern "C" fn free_func(_contents: *mut c_void, free_user_data: *mut c_void) {
-        let _ = Arc::from_raw(free_user_data as _);
+        let _ = Arc::from_raw(free_user_data as *const _);
```

Comment:
What was the warning, why is `*const _` better than `_`?

Reply:
I used *const because from_raw receives `*const `
```
pub unsafe fn from_raw(ptr: *const T) -> Self {
        unsafe { Arc::from_raw_in(ptr, Global) }
    }
```
the warning was:
```
warning: creating a `Arc` from a void raw pointer
   --> components/script/dom/bindings/buffer_source.rs:421:17
    |
421 |         let _ = Arc::from_raw(free_user_data as _);
    |                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    |
help: cast this to a pointer of the appropriate type
   --> components/script/dom/bindings/buffer_source.rs:421:31
    |
421 |         let _ = Arc::from_raw(free_user_data as _);
    |                               ^^^^^^^^^^^^^^^^^^^
    = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#from_raw_with_void_ptr
    = note: `#[warn(clippy::from_raw_with_void_ptr)]` on by default
```